.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
It is recommended to avoid hardcoding sensitive information such as API tokens directly in the code. Instead, use environment variables or a secure configuration management system to manage these secrets safely.